### PR TITLE
enable ckan cache

### PR DESCRIPTION
**Pull request by FuhuXia**
Part of https://github.com/GSA/data.gov/issues/3887

This will make ckan to response with `cache-control: public` instead of `private`, so most pages will cached by CDN.

https://docs.ckan.org/en/2.9/maintaining/configuration.html#ckan-cache-enabled

### CODE REVIEW

File: ckan/setup/ckan.ini [hunk 1]
```diff
@@ -42,6 +42,7 @@ beaker.session.timeout=900
 
 # CKAN caching
 ckan.cache_expires = 86400
```
Comment (jbrown-xentity) [inline ckan/setup/ckan.ini:44]: Caching pages for 24 hours seems agressive; do you think we could turn that down to maybe 1 hour?

**Review by jbrown-xentity — DISMISSED** [review]

File: ckan/setup/ckan.ini (hunk 1 above)
Comment (FuhuXia) [reply ckan/setup/ckan.ini:44]: 1 hour expiry will increase server load. Let us play safe and go with default value first. We can re-evaluate a good expiry after things are running on cloud.gov after a while.

Comment (FuhuXia) [reply ckan/setup/ckan.ini]: based on preliminary load tests on development space, it is safe to use 1 hour cache time instead of 1 day.

**Review by nickumia-reisys — APPROVED** [review]
🐐

**Pull request merged**